test/indexer/test_hca_indexer.py

```diff
@@ -814,6 +814,17 @@ def test_cell_line_sample(self):
                 self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (child_cell_line)'])
                 self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [config.null_keyword])
 
+    def test_file_core(self):
+        """
+        """
+        self._index_canned_bundle(('e0ae8cfa-2b51-4419-9cde-34df44c6458a', '2018-12-05T230917.591044Z'))
```

Comment:
If we are adding a new test, which requires indexing the bundle, I'd rather can a new bundle from production that has this field filled out so we don't need to modify an existing can.